Fix
> ValueError: not enough values to unpack (expected 4, got 0)

See https://github.com/pytorch/pytorch/pull/183668 & https://github.com/pytorch/pytorch/pull/189415

Author: Alexander Grund (TU Dresden)

diff --git a/test/test_scaled_matmul_cuda.py b/test/test_scaled_matmul_cuda.py
--- a/test/test_scaled_matmul_cuda.py
+++ b/test/test_scaled_matmul_cuda.py
@@ -1748,6 +1748,7 @@ class TestFP8Matmul(TestCase):
         self.assertEqual(out_fp32, out_fp8.to(torch.float))
 
     @skipIfRocm(msg="https://github.com/pytorch/pytorch/issues/164271")
+    @onlyCUDA
     @unittest.skipIf(IS_WINDOWS, "Windows doesn't support row-wise scaling")
     @unittest.skipIf(not PLATFORM_SUPPORTS_FP8, f8_msg)
     @unittest.skipIf(not SM90OrLater, "sm89 kernel isn't opted into carveout yet")
@@ -1761,6 +1761,11 @@ class TestFP8Matmul(TestCase):
         x_fp8 = to_fp8_saturated(x / x_scales, e4m3_type)
         y_fp8 = to_fp8_saturated(y / y_scales, e4m3_type)
 
+        # Warm up so hipBLASLt's one-time init kernel does not appear in the
+        # profile trace below.
+        scaled_mm_wrap(x_fp8, y_fp8, scale_a=x_scales, scale_b=y_scales, out_dtype=torch.bfloat16)
+        torch.cuda.synchronize()
+
         cu_count = torch.cuda.get_device_properties().multi_processor_count
         carveout = 66 if torch.version.cuda else cu_count // 8

